DESTDIR doesnt require clean up of things outside of DESTDIR

--- INSTALL
+++ INSTALL
@@ -7,7 +7,7 @@
 SHRDIR=$DESTDIR/usr/share/collectl
 MANDIR=$DESTDIR/usr/share/man/man1
 ETCDIR=$DESTDIR/etc
-INITDIR=$DESTDIR/$ETCDIR/init.d
+INITDIR=$ETCDIR/init.d
 
 mkdir -p $BINDIR
 mkdir -p $DOCDIR
@@ -46,6 +46,8 @@
 chmod 755 $SHRDIR/util/*
 chmod 444 $SHRDIR/formatit.ph $SHRDIR/*expr.ph $SHRDIR/vmstat.ph
 
+[[ ${DESTDIR} != "/" ]] && exit 0
+
 # remove any stale versions in case the names/numbers used have changed.
 # on new ROCKS installion 'rm' isn't there yet!  [thanks roy]
 if [ -x /bin/rm ] ; then
